vocabularies/UI.xml

```diff
@@ -1291,7 +1291,10 @@ The trend is
         <Annotation Term="Validation.Pattern" String="^[+-]?[0-9][0-9]:[0-5][0-9]$" />
         <Annotation Term="Core.Description" String="Display the annotated timestamp field in this time zone" />
         <Annotation Term="Core.LongDescription">
-          <String>The display time zone is represented as an offset ±hh:mm and can be a path expression</String>
+          <String>The display time zone is represented as an offset ±hh:mm and can be a path expression.
+If a path expression is used to reference a field containing the numeric offset,
+this offset field can in turn be annotated with `Common.Text` pointing to a third field
+containing a textual description of the display time zone.</String>

```

Comment:
Why do you need both a textual description and the timezone offset `+/-hh:mm`? The timezone offset can be transported with the `Edm.DateTimeOffset` property itself, as proposed in #68.

Reply:
@RaizoC what do you think about placing the offset in the timestamp value?

@HeikoTheissen since there is no definitive code list for symbolic time zones a display timezone such as `CST` cannot be used for calculations in the client. The timestamp would have to be displayed "as is", trusting that the numeric offset in the DateTimeOffset value matches the symbolic timezone in the DisplayTimeZone annotation.

Reply:
@ralfhandl I don't think we should put some semantics into the offset of the Edm.DateTimeOffset content as this might be misleading regarding its interpretation (and would also be incompatible to introduce such a semantics afterwards). Up to now the value is just a representation of one and the same timestamp without any semantics defined. I like the approach with a dedicated annotated property as it also allows to use it accordingly for filtering, sorting, etc. Further if we would have the need to add further semantics / reasonable display time zones to the timestamp we could easily do that (e.g. timestamp in sending and receiving location or timestamp in start and destination airport.

Reply:
> @HeikoTheissen since there is no definitive code list for symbolic time zones a display timezone such as `CST` cannot be used for calculations in the client.

I doubt that any smartphone would function if there wasn't a standardized time zone database. See, for example, https://github.com/eggert/tz/blob/master/europe#L1490 for the definition of Europe/Berlin. The same file also defines (on line #L743) CET "for backward compatibility".

And https://github.com/eggert/tz/blob/master/northamerica#L204 defines CST6CDT, but CST does not appear as a time zone ID.

Reply:
@HeikoTheissen Maybe this will work: https://www.iana.org/time-zones

Please tell me what you find out.

Reply:
The eggert/tz repository is mentioned on https://data.iana.org/time-zones/tz-link.html.